lib/sass_spec/test.rb

```diff
@@ -30,6 +30,8 @@ def run_spec_test(test_case, options = {})
     return unless handle_expected_error_message!(test_case, options)
     return unless handle_unexpected_error_message!(test_case, options)
   end
+
+  return true
```

Comment:
indicates that the test case passed.
We could probably store result to test_case here